i18n/locales/source/animations/spritelab_animation_library.json

```diff
@@ -48,7 +48,6 @@
   "angry": "angry",
   "animal": "animal",
   "animals": "animals",
-  "ant": "ant",
```

Comment:
he had hiiiiiiiiiiiigh hopes.
he had hiiiiiiiiiigh hopes.
he had high apple-pie-in-the-skyyyyy hopes
So any time you're gettin' low
'Stead of lettin' go
Just remember that ant
oops there goes another... spritelab animation

These are a slight mystery. They get populated by looking at s3 buckets and pulling out the animation metadata there and are not locally derived in the repo. it is hard to go "ah, yes, that makes sense. there's no 'ant' animation anymore"

this sync up looks a lot better. 9 files is better than 900. seeing lots of cool accessibility stuff. 👍